core/conversion/converters/impl/einsum.cpp

```diff
@@ -0,0 +1,62 @@
+#include "NvInfer.h"
+#include "core/conversion/converters/converters.h"
+#include "core/conversion/tensorcontainer/TensorContainer.h"
+#include "core/util/prelude.h"
+#include "torch/torch.h"
+
+#include <ATen/ATen.h>
+#include <vector>
```

Comment:
Want to verify these imports and see if any are redundant or not needed

Reply:
core/util/prelude.h brings in NvInfer.h, ATen/Aten.h and torch/torch.h